reporter_test.go

```diff
@@ -0,0 +1,21 @@
+package httpexpect
+
+import (
+	"testing"
+)
+
+type mockT struct {
+	testing.T
+}
```

Comment:
I suggested to wrap testing.T in a struct to redefine its Fatalf method. But you implemented the test without redefining a method (which is a better approach), so wrapping it into a struct is not needed too, we can just create an instance of testing.T directly.